.gitignore

```diff
@@ -6,10 +6,10 @@ coverage/
 *.pb
 .idea/
 .nyc_output
-yarn.lock
-package-lock.json
```

Comment:
It is atypical for libraries to ship a `package-lock.json`. Why remove it here?

blocking on clarity or change

Reply:
Not sure why I removed these awhile back. Re-added.